web/controllers/wager_controller.ex

```diff
@@ -0,0 +1,68 @@
+defmodule MovieWagerApi.WagerController do
+  use MovieWagerApi.Web, :controller
+
+  alias MovieWagerApi.{Wager, WagerSerializer}
+
+  def create(conn, %{"data" => %{"attributes" => wager_params, "relationships" => relationship_params}}) do
+    params = get_relationship_ids(relationship_params)
+      |> Map.merge(wager_params)
+
+    changeset = Wager.changeset(%Wager{}, params)
+
+    case Repo.insert(changeset) do
+      {:ok, wager} ->
+        serialized_wager(conn, wager, :created)
+      {:error, changeset} ->
+        if changeset.errors[:user_movie_round] do
+          send_resp(conn, :unprocessable_entity, "You have already created a bet for this round")
+        else
+          send_resp(conn, :unprocessable_entity, "Invalid entry")
+        end
+    end
+  end
+
+  def index(conn, %{"user_id" => user_id, "movie_round_id" => movie_round_id}) do
+    query = from(w in Wager, where: [user_id: ^user_id], where: [movie_round_id: ^movie_round_id])
```

Comment:
On second thought, perhaps you can remove the `preload()` here, as it looks like `serialized_wager/3` will call `preload()`.